src/binaryen-c.h

```diff
@@ -103,6 +103,7 @@ typedef void* BinaryenFunctionTypeRef;
 // Add a new function type. This is thread-safe.
 // Note: name can be NULL, in which case we auto-generate a name
 BinaryenFunctionTypeRef BinaryenAddFunctionType(BinaryenModuleRef module, const char* name, BinaryenType result, BinaryenType* paramTypes, BinaryenIndex numParams);
+BinaryenFunctionTypeRef BinaryenGetFunctionType(BinaryenModuleRef module, BinaryenType result, BinaryenType* paramTypes, BinaryenIndex numParams);
```

Comment:
My issue is just that I am already using this functionality, but you are right, it's mostly because of the lack of an alternative. What if we'd add this for now, with a name different than `GetFunctionType` as you suggested, and I add a comment that it is subject to be removed again once a better alternative is in place?